spec/seoshop-api/api/tax_spec.rb

```diff
@@ -24,7 +24,7 @@
   end
 
   it '#get_default_tax' do
-    expect(subject).to receive(:get_taxes) { taxes_response }
+    subject.stubs(:get_taxes).returns(taxes_response)
```

Comment:
i don't think it's a good idea. It will be look like two coding styles are mixed, rspec tests but then also not. It's better to only use the expect style for consistency reasons.